src/Uno.UWP/Generated/3.0.0.0/Windows.Media.SpeechRecognition/SpeechRecognitionCompilationResult.cs

```diff
@@ -1,22 +0,0 @@
-#pragma warning disable 108 // new keyword hiding
```

Comment:
Some of the removed files will "activate" their classes for NET46 and WASM too... is it wanted? Because they won't be marked as not implemented for those platforms.

Reply:
NET46 and WASM implementation will be for POCO or enum only. It's expected